### DISCUSSION TITLE
Example of pyquesta and sv_conduit?

### DISCUSSION THREAD

**Discussion by dungnguyenanh** [raysalemi/Python4RTLVerification#7] [General]
Hi Ray,

I found pyquesta and sv_conduit quite interesting.
I try to follow README of pyquesta package as well as cocotb issue [#2638](https://github.com/cocotb/cocotb/issues/2638).

However, I got the following error

```
pyquesta.py", line 58, in get 
      cls.c_lib.get.restype = ctypes.c_char_p
      AttributeError: 'NoneType' object has no attribute 'get'

```
It seems that pyquesta can't find the sv_conduit.so library (even though I compiled it manually).

Can you give me some hints how to fix that?
If possible, could you please share an example that works with TinyALU design?

Thank you.

**Comment by raysalemi** [comment]
You’d have to ask your Mentor AE. I don’t have access to pyquesta anymore.

Did you use the -sc_lib option to name the .so file?

**Comment by dungnguyenanh** [comment]
Hi Ray,
Thanks, I got it.
Nope, I refer to the pyquesta.mk file which use gcc to compile the .so file.
